Meta/build-image-qemu.sh

```diff
@@ -31,8 +31,22 @@ if [ "$(id -u)" != 0 ]; then
     if [ -x "$FUSE2FS_PATH" ] && $FUSE2FS_PATH --help 2>&1 |grep fakeroot > /dev/null; then
         USE_FUSE2FS=1
     else
-        ${SUDO} -E -- "$0" "$@" || die "this script needs to run as root"
-        exit 0
+        # we don't want to exit on error here,
+        # to be able to handle sudo exit status
+        set +e
+
+        ${SUDO} -E -- sh -c "\"$0\" $* || exit 42"
```

Comment:
The comment doesn't follow the usual comment style, but I'd argue that it's superfluous anyways, so...

```suggestion
        set +e
        ${SUDO} -E -- sh -c "\"$0\" $* || exit 42"
```

Reply:
removed the comment